connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -23,7 +26,52 @@ class UplynkConnector(
     private val theoplayerView: THEOplayerView,
 ) {
     private lateinit var integration: UplynkAdIntegration
-    private val eventDispatcher = UplynkEventDispatcher()
+
+    /**
+     * Scheduled ad breaks
+     */
+    var adBreaks: List<UplynkAdBreak>? = null
+        private set
+
+    /**
+     * The ad break that is currently played or `null` otherwise
+     */
+    var currentAdBreak: UplynkAdBreak? = null
+        private set
+
+    /**
+     * The ad that is currently played or `null` otherwise
+     */
+    var currentAd: UplynkAd? = null
+        private set
```

Comment:
Maybe more of a philosophical question: why do we have both `UplynkConnector.currentAds` and `player.ads.currentAds`? Wasn't the whole point of the custom SSAI integrations that we can put everything behind the THEOplayer ads API?

Same thing with the new `UplynkListener` methods: how are those different from the existing events on `player.ads`?

Reply:
The only difference between them is the type of Ads object. `player.ads.currentAds` - has common type of an ad, that is part of THEOplayer project. `UplynkConnector.currentAds` has specific type for Uplynk ads. The same as for `UplynkListener`

I think we have to discuss if we even need this PR or if we could find some other solution

Reply:
Okay, I think this is a limitation of our custom SSAI API then. We should provide a way to add some integration-specific data to an custom `Ad` or `AdBreak`.

I suggest we park this PR for now until we have that new API.